src/main/kotlin/com/hibob/academy/service/SessionService.kt

```diff
@@ -18,9 +18,8 @@ class SessionService {
     fun createJwtToken(user: User): String {
         return Jwts.builder()
             .setHeaderParam("typ", "JWT")
-            .claim("email", user.email)
-            .claim("user name", user.userName)
-            .claim("isAdmin", user.isAdmin)
+            .claim("employeeId", user.employeeId)
+            .claim("companyId", user.companyId)
```

Comment:
this is ok for now. eventually we're gonna need to verify that this company id and employee id exists in the db, and get the role from the db and add it as a claim